packages/carousel/src/react/index.js

```diff
@@ -179,24 +175,6 @@ function Instructions(props) {
 
 const Pages = React.forwardRef((props, ref) => {
   const context = React.useContext(CarouselContext)
-  const prevActivePage = usePrevious(context.activePage)
-  const [transitioning, setTransitioning] = React.useState(false)
```

Comment:
we shouldn't need the transition group anymore. we could remove that dependency

Reply:
Hmm, wait, I still see `Transition` used in `carousel/src/react/index.js`, line 217.  Isn't that still needed to support the page advance slide animation?

Reply:
i added the Transition so i could hide items after the transition. with the overflow we shouldn't need to use Transition.